core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -290,6 +290,12 @@ class RequestChannel(val queueSize: Int, val metricNamePrefix : String) extends
     }
   })
 
+  def totalSelectorChannelCount: Int = {
```

Comment:
Since this is only used for testing, we could remove this and use `TestableSocketServer` in the test instead.